client/flutter/lib/pages/latest_numbers.dart

```diff
@@ -20,118 +20,158 @@ const name = TextStyle(
-              builder: (context, snapshot) {
-                final hasGlobalStats =
-                    snapshot.hasData && snapshot.data['globalStats'] != null;
-                final globalStats =
-                    hasGlobalStats ? snapshot.data['globalStats'] : null;
-                final ts = hasGlobalStats
-                    ? DateTime.fromMillisecondsSinceEpoch(
-                        globalStats['lastUpdated'])
-                    : DateTime.now();
-                final numFmt = NumberFormat.decimalPattern();
-                final lastUpd = DateFormat.MMMMEEEEd().add_jm().format(ts);
-                return SliverList(
-                    delegate: SliverChildListDelegate([
-                  StatCard(
-                      background: LatestNumbersGraph(
-                        timeseries:
-                            hasGlobalStats ? globalStats['timeseries'] : null,
-                        timeseriesKey: 'dailyCases',
-                      ),
-                      title: Text(
-                        S.of(context).latestNumbersPageGlobalCasesTitle,
-                        style: name,
-                      ),
-                      content: Text(
-                        hasGlobalStats && globalStats['cases'] != null
-                            ? numFmt.format(globalStats['cases'])
-                            : '-',
-                        softWrap: true,
-                        style: hasGlobalStats && globalStats['cases'] != null
-                            ? number
-                            : loadingStyle,
-                        textAlign: TextAlign.left,
-                      )),
-                  StatCard(
-                    background: LatestNumbersGraph(
-                      timeseries:
-                          hasGlobalStats ? globalStats['timeseries'] : null,
-                      timeseriesKey: 'dailyDeaths',
-                    ),
-                    title: Text(S.of(context).latestNumbersPageGlobalDeaths,
-                        style: name),
-                    content: Text(
-                      hasGlobalStats && globalStats['deaths'] != null
-                          ? numFmt.format(globalStats['deaths'])
-                          : '-',
-                      softWrap: true,
-                      style: hasGlobalStats && globalStats['deaths'] != null
-                          ? number
-                          : loadingStyle,
-                      textAlign: TextAlign.left,
-                    ),
-                  ),
-                  Container(
-                    height: 25,
-                  ),
-                  Text(
-                    snapshot.hasData
-                        ? S.of(context).latestNumbersPageLastUpdated(lastUpd)
-                        : S.of(context).latestNumbersPageUpdating,
-                    style: TextStyle(color: Color(0xff26354E)),
-                    textAlign: TextAlign.center,
+          CupertinoSliverRefreshControl(
```

Comment:
The `?` that shows up on the gesture before refresh is triggered feels a little weird to me. Can we make this something more standard? Obviously this comment is moot if we're removing pull-to-refresh